packages/cfd/src/features/Components/__tests__/copy-text-icon.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { CopyTextIcon } from '../copy-text-icon';
+
+describe('<CopyTextIcon />', () => {
+    it('component should be rendered', () => {
+        const { getByTestId } = render(<CopyTextIcon />);
+        const mainDiv = getByTestId('cfd_account_copy_main_div');
```

Comment:
since the purpose of the PR is to remove the prefixes, better to remove it here as well
```suggestion
        const mainDiv = getByTestId('account_copy_main_div');
```